packages/calcite-components/src/components/handle/handle.scss

```diff
@@ -9,7 +22,8 @@
     justify-center
     self-stretch
     border-none;
-  color: theme("borderColor.color.input");
+
+  color: var(--calcite-handle-icon-color, var(--calcite-color-border-input));
```

Comment:
@ashetland can you confirm `--calcite-color-border-input` should be set as a icon color for the handle component?

Reply:
Yes that's correct.